lib/test2.js

```diff
@@ -98,7 +98,7 @@ context.proxy = {
 context.rootServices = rootServices;
 const webServer = makeWebServer();
 if (!webServer.isConfigValid(configData)) {
-  throw new Error("config invalid")
+  throw new Error("ZWED0048E - config invalid")
```

Comment:
This is a test file so please note that it's never going to be seen.
In fact, I'm unsure if this test is being used.

Reply:
corrected.